xep-0272.xml

```diff
@@ -24,7 +24,7 @@
   </dependencies>
   <supersedes/>
   <supersededby/>
-  <shortname>muji</shortname>
+  <shortname>Muji</shortname>
```

Comment:
Don't we typically use lowercase shortnames?

Reply:
I saw that at on https://xmpp.org/extensions/xep-0384.html#appendix-docinfo and https://xmpp.org/extensions/xep-0369.html#appendix-docinfo. But https://xmpp.org/extensions/xep-0313.html#appendix-docinfo has indeed lowercase letters. @horazont Is that documented somewhere?

Reply:
We indeed typically use lowercase names, I think, but that's not documented anywhere. Could you please fix that?

Reply:
OK, thanks. I thought that the short name would simply be the name that is used instead of the complete name *Multiparty Jingle*. That would be *Muji* (and not *muji*) as the title says. But if that is not the case, in which context is the short name used then?

Reply:
I just checked through the list of all shortnames that we have and all of them use either uppercase or lowercase letters, combined with numerics and/or hyphens, except for two:
- XEP-0106 has shortname `jid\20escaping` - I'd consider this exception acceptable ;)
- XEP-0372 has shortname `Refs` - even worse than what we're discussing here IMO

That said, I don't think there is any real use of the shortname so it also doesn't really matter. I'd rather like to have a "XEP URN", which should be the same as the "primary" XML namespace used in the XEP. Often enough that namespace already is "urn:xmpp:$shortname:$version". But that's a whole different topic.